x-pack/plugins/case/server/client/client.ts

```diff
@@ -219,8 +219,8 @@ export class CaseClientHandler implements CaseClient {
     } catch (error) {
       throw createCaseError({
         message: `Failed to get alerts using client ids: ${JSON.stringify(
-          args.ids
-        )} \nindices: ${JSON.stringify([...args.indices])}: ${error}`,
+          args.alertsInfo
```

Comment:
is it `alerts` or `alertsInfo`...Looking at your other change in this file on line 172. Also, forgot to change the "ids" to "alerts" right before the stringify in this change

Reply:
Oh thank you. Yeah `alertsInfo` vs `alerts` is kind of confusing. They're slightly different structures but I didn't know how to name them haha. Any ideas?